### PR TITLE
💄Email block redesign #623

### LINKED ISSUE #623: :art:  Email Block Redesign: block redesign
## User stories

As a user, I want the "Email Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/email-message-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/email-message-block/`.

### PR DESCRIPTION
**Pull request by PerisOduol618**
# Description

The "Email Block" component in the app is redesigned to enhance its usability and visual appeal.

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes # (issue) - *Example*: Fixes #623

# Screenshot (optional)
image

image

# Checklist:

- [ X] My code follows the style guidelines of this project
- [ X] I have performed a self-review of my code
- [X ] I have commented my code, particularly in hard-to-understand areas
- [X ] I have made corresponding changes to the documentation

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/email-block.jsplumb.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export function _EmailBlockDecoratePlumb(block: EmailMessageBlock, comp: Compone
     target: true,
     cssClass:"block_endpoint",
     endpoint: "Dot",
-    anchor: "Left",
+    anchor: [0, 0.23, 1, 0],
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/email-block.jsplumb.ts:25]: Let's leave this as "Left" for now

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Sync then resolve conflicts kindly.

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/email-block.jsplumb.ts (hunk 1 above)
Comment (PerisOduol618) [reply libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/email-block.jsplumb.ts]: @LemmyMwaura Done

**Comment by LemmyMwaura** [comment]
Closing this as it's been resolved by Dennis. His implementation was first but good work

**Pull request closed without merging**